common/changes/@bentley/tree-widget-react/classifications-tree_2020-07-13-09-00.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
@calebmshafer we won't need this patch changelog right?

Reply:
You can keep both if you'd like so both "functional component" and the "Add classifications tree" show up in the changelog.

It shouldn't hurt anything, Rush should use the highest one available..